_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -26,101 +26,34 @@ To enable searchable snapshots, reference the [instructions](https://opensearch.
+OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
 
-For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy, removing the least recently used segments first.
+For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy.
 
 ## Performance
 
-When cached data is searched repeatedly, searchable snapshots provide consistent search performance and throughput. Performance is slow the first time the snapshot data is accessed, but subsequent searches are comparable in performance to regular OpenSearch cluster searches. Note, however, that the ratio of cache size to the combined size of the searchable snapshot, along with the search pattern, will factor into performance. The more data the cache misses, the lower the overall performance. In the next section, we cover the performance results of our benchmark testing.
+Comparing the performance of searchable snapshot queries and local index queries on the StackOverflow dataset, the initial run of Searchable Snapshots took approximately twice as long as local index. However, subsequent queries showed similar performance for both searchable snapshots and local index because searchable snapshots used cached data.
 
 Note that searchable snapshots cannot be deleted until the `searchable` flag on these snapshots is removed. Follow the snapshot [configuration steps]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/) in order to remove the flag.
 
 ## Benchmarks
 
-For the following benchmarks, the OpenSearch benchmark workload was configured with 0 warmup iterations so that the cache was not populated when the measurements started. The test aimed to achieve 1.5 operations per second. `search_clients` was configured to 16 to match the number of CPU cores on the hosts. For searchable snapshots, the test was performed immediately after creating the searchable snapshot index. For UltraWarm, the test was performed immediately after migrating the index to the warm tier so that the cache was minimally warmed in both test cases.
-
-<table><colgroup><col /><col /><col /><col /><col /></colgroup>
```

Comment:
The formatting is confusing to read. Can we add grid lines and make it more clear?